frontend/packages/ux-editor/src/hooks/useSelectedFormLayoutSetName.ts

```diff
@@ -1,22 +1,25 @@
 import { useStudioUrlParams } from 'app-shared/hooks/useStudioUrlParams';
 import { useLayoutSetsQuery } from 'app-shared/hooks/queries/useLayoutSetsQuery';
-import { useReactiveLocalStorage } from 'app-shared/hooks/useReactiveLocalStorage';
+import { useEffect, useState } from 'react';
 
 export type UseSelectedFormLayoutSetNameResult = {
   selectedFormLayoutSetName: string;
   setSelectedFormLayoutSetName: (layoutName: string) => void;
-  removeSelectedFormLayoutSetName: () => void;
 };
 
 export const useSelectedFormLayoutSetName = (): UseSelectedFormLayoutSetNameResult => {
   const { org, app } = useStudioUrlParams();
   const { data: layoutSets } = useLayoutSetsQuery(org, app);
 
-  const [
-    selectedFormLayoutSetNameState,
-    setSelectedFormLayoutSetName,
-    removeSelectedFormLayoutSetName,
-  ] = useReactiveLocalStorage('layoutSet/' + app, undefined);
+  const savedLayoutSetName = localStorage.getItem('selectedFormLayoutSetName');
+  const [selectedFormLayoutSetNameState, setSelectedFormLayoutSetName] =
+    useState(savedLayoutSetName);
```

Comment:
Based on the two previous comments, we can now simplify this one line like this:
```suggestion
const [selectedFormLayoutSetNameState, setSelectedFormLayoutSetName] =
    useState(typedLocalStorage.getItem<string>(storageKey));
```

Reply:
fixed